### PR TITLE
feat - Push default terms behind a flag

### PR DESCRIPTION
**Pull request by bebraw**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -66,6 +67,13 @@ function reporter(context, options = {}) {
 	};
 }
 
+function getDefaultTerms(opts) {
```
Comment (sapegin) [inline index.js]: It actually returns all terms, not just default ;-)

Comment (bebraw) [inline index.js]: Yeah, I'll rename.

Comment (bebraw) [inline index.js]: Done. Also tidied up a bit.

**Comment by sapegin** [comment]
:shipit:

**Pull request merged**